__tests__/utils/tokenVerify.test.js

```diff
@@ -1,20 +1,39 @@
-import tokenVerify from "../../src/Utils/tokenVerify";
+import tokenVerify, { getTokenFromParams } from "../../src/Utils/tokenVerify";
 
 describe(" TokenVerify test suite", () => {
-  it("should return false if no or wrong token is passed", () => {
-    const returned = tokenVerify();
-    expect(returned).toBe(false);
+  describe(" Verifies token in Sign In", () => {
```

Comment:
Put **Local Sign In Token Verification**

Reply:
Feedback implemented